### ISSUE TITLE
[Design Deliverable] Update Project Updates site with Oct-2.2 content

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#2298]
**Description:**

Adds topline synthesis content covering October-Pilot 2.2 to Project Updates site. This should largely be a matter of adding any applicable visuals to some high level excerpts from the written synthesis. This will be presented to regional staff

**AC:**
- [ ] The design is usable and accessible, meaning it adheres to definition of done standards for design work.
> <details>
>      <summary>Expand Definition of Done Standards</summary>
> 
> - It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented
> 
> - Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented
> - It follows [accessibility guidelines](https://accessibility.digital.gov/) and accessibility implementation notes are documented (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.)
> - If feedback identifies bigger questions or unknowns, create additional issues to investigate
> </details>

**Tasks:**

- [ ] Create a branch in Project Updates / Knowledge Center test environment
- [ ] Add any new imagery to a descriptively titled subfolder of img in the repo
- [ ] Pull relevant synthesis snippets
- [ ] Draft new HTML page
- [ ] Update nav on all pages
- [ ] Make any available version bumps to site scripts
- [ ] Open PR to raft-tech and tag Thomas & Alex for a11y/QASP review

**Supporting Documentation:**

—Link to Figma and Dev-Ready Design folder PDF—

**Comment by ADPennington** [comment]
fyi: the briefing has been scheduled for 1/19/2023.

**Comment by reitermb** [comment]
@ADPennington Sorry for the delay on this! Blocker to the [PR](https://github.com/raft-tech/TANF-app/pull/2339) should be fixed now though & everything is live in the [test ](https://reitermb.github.io/Test/) environment

**Comment by lfrohlich** [comment]
@reitermb Found a few typos, otherwise content looks good to me. Will wait to approve until Thomas has reviewed for 508. 
* Typo in the left side nav (Januarry) 
* Last checkmark (Research as an onboarding aide): "One however has been taking the **opportunites**..." "...what we're asking of them in the **rsearch**"
* Last paragraph: "...where it's **occuring** in their data"

**Comment by reitermb** [comment]
> @reitermb Found a few typos, otherwise content looks good to me. Will wait to approve until Thomas has reviewed for 508.
> 
> * Typo in the left side nav (Januarry)
> * Last checkmark (Research as an onboarding aide): "One however has been taking the **opportunites**..." "...what we're asking of them in the **rsearch**"
> * Last paragraph: "...where it's **occuring** in their data"

Thanks @lfrohlich! Fixes for those should be live now

**Comment by ADPennington** [comment]
@reitermb thanks for this. some review notes below. we can discuss today.

> ### Who participated in the expansion

The visual is a bit hard to follow. what do we want the audience to know? i think they will mostly want to know how many grantees (and what type) were recruited vs participated from a given region. the 477 status might also be useful descriptor but not necessary. the q may come up. 


> ### What we tested

I'm not sure folks will remember what we tested last time. a bulleted list could be helpful here. the changelog for 2.0 doesnt include all the features. 

> ### Our findings

do we think the audience will understand these findings without any sort of visual? i dont think we can expect folks to read this text during the presentation or read the full synthesis and im not sure those less familiar will be able to visualize how these issues happen. i think we need screenshots but not sure of a11y implications. cc: @ttran-hub 


> ###  Initial findings

can you replace this heading with "initial error-related research findings"?


### other stuff

- is there a reason we havent called out the next expansion and the features coming next? seems important, but maybe there's good reason not to be too specific
- seems like we're missing the positive feedback examples and what we heard in terms of features for future consideration like submission log

**Comment by reitermb** [comment]
Pushed an update addressing feedback areas—deploying to the test environment shortly! All the additions were made using components that have already passed a11y review—always welcome if we want to do another pass but its compliance should be unchanged. 

cc: @ADPennington @lfrohlich

**Comment by stevenino** [comment]
@ADPennington @lfrohlich Are we good to close this tix?

**Comment by lfrohlich** [comment]
@reitermb for ease of re-review, could you respond to alex's comments here? Or is there somewhere else where it would be easy to see what has changed in response to her comments? Thanks

**Comment by reitermb** [comment]
> > ### Who participated in the expansion
> 
> The visual is a bit hard to follow. what do we want the audience to know? i think they will mostly want to know how many grantees (and what type) were recruited vs participated from a given region. the 477 status might also be useful descriptor but not necessary. the q may come up.

Added breakdown of userbase by type/region below chart
 
> > ### What we tested
> 
> I'm not sure folks will remember what we tested last time. a bulleted list could be helpful here. the changelog for 2.0 doesnt include all the features.

Added recap of core 2.0 functionality above newly tested for 2.1/2.2 

> > ### Our findings
> 
> do we think the audience will understand these findings without any sort of visual? i dont think we can expect folks to read this text during the presentation or read the full synthesis and im not sure those less familiar will be able to visualize how these issues happen. i think we need screenshots but not sure of a11y implications. cc: @ttran-hub

Added visuals to each finding (including alt-text & lightboxes)

> > ### Initial findings
> 
> can you replace this heading with "initial error-related research findings"?

Replaced heading

> ### other stuff
> * is there a reason we havent called out the next expansion and the features coming next? seems important, but maybe there's good reason not to be too specific

The main reason was uncertainty about which features would make the cut when the bulk of this update was drafted in December. I think we can definitely speak to submission history / visualize that a bit via a quick demo in Figma though. I was going to field some preferences on that at UX sync tomorrow.

> * seems like we're missing the positive feedback examples and what we heard in terms of features for future consideration like submission log

Added positive feedback to main findings section

cc: @lfrohlich @ADPennington

**Comment by lfrohlich** [comment]
The visuals are a great add

**Comment by ADPennington** [comment]
> > > ### Who participated in the expansion
> > 
> > 
> > The visual is a bit hard to follow. what do we want the audience to know? i think they will mostly want to know how many grantees (and what type) were recruited vs participated from a given region. the 477 status might also be useful descriptor but not necessary. the q may come up.
> 
> Added breakdown of userbase by type/region below chart


can you double-check this? VI is in region 2 which is not referenced here. 
> 
> > > ### What we tested
> > 
> > 
> > I'm not sure folks will remember what we tested last time. a bulleted list could be helpful here. the changelog for 2.0 doesnt include all the features.
> 
> Added recap of core 2.0 functionality above newly tested for 2.1/2.2

lgtm! 
> 
> > > ### Our findings
> > 
> > 
> > do we think the audience will understand these findings without any sort of visual? i dont think we can expect folks to read this text during the presentation or read the full synthesis and im not sure those less familiar will be able to visualize how these issues happen. i think we need screenshots but not sure of a11y implications. cc: @ttran-hub
> 
> Added visuals to each finding (including alt-text & lightboxes)

lgtm! 

> > > ### Initial findings
> > 
> > 
> > can you replace this heading with "initial error-related research findings"?

lgtm! 

> Replaced heading
> 
> > ### other stuff
> > 
> > * is there a reason we havent called out the next expansion and the features coming next? seems important, but maybe there's good reason not to be too specific
> 
> The main reason was uncertainty about which features would make the cut when the bulk of this update was drafted in December. I think we can definitely speak to submission history / visualize that a bit via a quick demo in Figma though. I was going to field some preferences on that at UX sync tomorrow.
> 
ok lets discuss at ux sync. 

> > * seems like we're missing the positive feedback examples and what we heard in terms of features for future consideration like submission log
> 
> Added positive feedback to main findings section

lgtm! 
> 
> cc: @lfrohlich @ADPennington

@reitermb back to you re: the userbase. this is in good shape to merge today. lets wrap up at ux sync if possible.

**Comment by reitermb** [comment]
Merged in the region 2 fix and separated the last 'what's next' into one for research as an onboarding aid (with slimmed down language) and one for 3.0 expansion (which we'll transition from into a mini-demo to highlight submission history). 

Ready for review! 

cc @ADPennington @lfrohlich

**Comment by lfrohlich** [comment]
Revised count for Region 4 prior to merge. Note, OFA needs some more revisions to the project updates, which will be handled in #2365

**Comment by lfrohlich** [comment]
Revised count for Region 4 prior to merge. Note, OFA needs some more revisions to the project updates, which will be handled in #2365
